UnitTests/HoldEmTests.cs

```diff
@@ -1,13 +1,315 @@
-﻿using System;
+﻿using CardsCashCasino.Manager;
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
+using System.Runtime.Versioning;
 using System.Text;
 using System.Threading.Tasks;
+using static CardsCashCasino.Manager.TexasHoldEmManager;
 
 namespace CardsCashCasino.NUnit
```

Comment:
These tests look spectacular. Way to go, dude. You really knocked these out of the park